megalinter/Linter.py

```diff
@@ -571,10 +571,10 @@ def load_config_vars(self, params):
                 )
             elif (
                 cli_lint_mode_descriptor == "file"
-                and cli_lint_mode_config == "list_of_files"
+                and (cli_lint_mode_config == "list_of_files" or cli_lint_mode_config == "project")
```

Comment:
nah, project can work as override of file mode ^^

Reply:
Done